File: app/src/main/java/com/github/jameshnsears/quoteunquote/configure/ConfigureActivityDouble.java
package com.github.jameshnsears.quoteunquote.configure;

import android.os.Bundle;

import androidx.annotation.Nullable;

import com.github.jameshnsears.quoteunquote.R;
import com.github.jameshnsears.quoteunquote.databinding.ActivityConfigureBinding;

import timber.log.Timber;

public class ConfigureActivityDouble extends ConfigureActivity {
    @Override
    protected void init() {
    }

    @Override
    public void onCreate(@Nullable final Bundle bundle) {
        Timber.d("onCreate");
        super.onCreate(bundle);

        widgetId = 1;

        activityConfigureBinding = ActivityConfigureBinding.inflate(this.getLayoutInflater());
        setContentView(activityConfigureBinding.getRoot());

        createListenerBottomNavigationView();

        activityConfigureBinding.configureNavigation.setSelectedItemId(R.id.navigationBarQuotations);
    }
}

File: app/src/manifestTest/java/com/github/jameshnsears/quoteunquote/configure/ConfigureActivityDouble.kt
package com.github.jameshnsears.quoteunquote.configure

import androidx.annotation.NonNull
import com.github.jameshnsears.quoteunquote.configure.fragment.content.ContentFragmentDouble

class ConfigureActivityDouble : ConfigureActivity() {
    override fun broadcastFinishIntent() {
        // don't broadcast anything for test(s)
    }

    @NonNull
    override fun getFragmentContentNewInstance(): ContentFragmentDouble {
        return ContentFragmentDouble.newInstance(widgetId)
    }
}
